app/controllers/users_controller.rb

```diff
@@ -18,11 +18,14 @@ def create
       render :new
     end
   end
-
+  
   private
-  def user_params
-    params.require(:user).permit(:name, :email, :address, :city, :state, :zip, :password, :password_confirmation)
-  end
+    def user_params
+      params.require(:user).permit(:name, :email, :address, :city, :state, :zip, :password, :password_confirmation)
+    end 
 
+    def user_update_params
```

Comment:
Only the `user/user_controller` needs the `user_update_params`. This `user controller` is only for new/unregistered users I believe so it require the `user_params` only